docs/src/pages/components/calendar.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Calendar
+group: Forms
+figma: https://www.figma.com/file/DFKyTGHAoDxOsUBPszLLxP/%F0%9F%8F%B5%EF%B8%8FMarigold?node-id=7064-19516&t=8m2OHdMLrDkvsV2P-0
+switchTheme: false
+badge: 'new'
+caption: Component for displaying and selecting date .
+---
+
+The `<Calendar>` is a date selection interface that allows you to choose date using calendar view .
+
+<Preview>
+  <BasicCalendar />
+</Preview>
+
+## Usage
+
+### Import
+
+To import the component you just have to use this code below.
+
+```tsx onlyCode
+import { Calendar } from '@marigold/components';
+```
+
+### Appearance
+
+<AppearanceTable component={title} />
+
+### Props
+
+<PropsTable
+  props={[
+    {
+      property: 'value',
+      type: 'DateValue',
+      description: 'The current calendar value.',
+      default: 'none',
+    },
+    {
+      property: 'defaultValue',
+      type: 'DateValue',
+      description: 'The default value of calendar.',
+      default: 'none',
+    },
+    {
+      property: 'focusedValue',
+      type: 'DateValue',
+      description: 'Controls the currently focused date within the calendar.',
+      default: 'none',
+    },
+    {
+      property: 'maxValue',
+      type: 'DateValue',
+      description: 'The maximum allowed date that a user may select.',
+      default: 'none',
+    },
+    {
+      property: 'minValue',
+      type: 'DateValue',
+      description: 'The minimum allowed date that a user may select.',
+      default: 'none',
+    },
+    {
+      property: 'validationState',
+      type: 'ValidationState `valid | invalid`',
+      description:
+        'Whether the current selection is valid or invalid according to application logic.',
+      default: 'none',
+    },
+    {
+      property: 'onChange',
+      type: 'function',
+      description:
+        'a callback function that is called when the the date field value changes.',
+      default: 'none',
+    },
+    {
+      property: 'disabled',
+      type: 'boolean',
+      description: 'Disables the Calendar.',
+      default: 'false',
+    },
+    {
+      property: 'readOnly',
+      type: 'boolean',
+      description: 'Whether the calendar value is immutable.',
+      default: 'false',
+    },
+    {
+      property: 'autoFocus',
```

Comment:
why does it have this as property? Shouldn't that be always the case? I was just wondering :D

Reply:
I agree with you , but may be they created it in case you used `Calendar` alone without a date picker , and there are some other components but you still want to focus `Calendar` component .